app/ide-desktop/lib/client/watch.ts

```diff
@@ -83,7 +85,11 @@ const ALL_BUNDLES_READY = new Promise<Watches>((resolve, reject) => {
         void dashboardBuilder.watch()
 
         console.log('Bundling content.')
-        const contentOpts = contentBundler.bundlerOptionsFromEnv()
+        const contentOpts = contentBundler.bundlerOptionsFromEnv({
+            devMode: true,
+            platform: common.Platform.desktop,
```

Comment:
1. I believe this is bad solution. We have logic in Rust like "if the platform was desktop and frame is hidden, show window control buttons in WebGL". Basically, platform "web" means a web browser. When using `run ... watch` you can run the app either in electron or in the browser and we want to know where it is being used. This setting should reflect that. If you want to control deep links, I'd advise to check the url if it is localhost instead.

2. The problem is not only with watch scripts. If you install our app via installer, you run it, you should still be able to connect to it from chrome, as Electron serves that on localhost. This way of running should be supported correctly as well.

Reply:
1. i think the desktop IDE uses localhost too (it has a server, after all) so i'm not sure whether that will work for deciding whether deep links are supported. i guess it's a good solution for whether to show or hide the local backend/remote backend switcher though.
it will be incorrect when doing `npm run watch-dashboard` though, but that should be more or less fine because:
  a) it only happens during development, not when deployed to `cloud.enso.org` and
  b) `127.0.0.1` can be used for that instead
2. hmm... maybe we can check for `'isElectron' in window` instead? (and define `isElectron = true` in preload?)
  there would still be issues with `./run ide watch` though, when using the electron window, since that doesn't support deep links. not sure if there are any good solutions to that